src/reporting/market.se

```diff
@@ -44,10 +45,13 @@ data tentativeWinningPayoutDistributionHash
 data finalPayoutDistributionHash
 data automatedReporterPayoutDistributionHash
 data automatedReporterDisputeBondHolder
+data automatedReporterDisputeBondToken
```

Comment:
We can delete all of these other dispute related variables as well.  Anyone code that accesses them should be updated to get the data from the dispute bond token (though I don't know that market needs any of that anymore).